docs/release-notes/self-hosted/self-hosted-v3.5.0.md

```diff
@@ -0,0 +1,72 @@
+# Self-hosted v3.5.0
+
+These release notes are for [Codacy Self-hosted v3.5.0](https://github.com/codacy/chart/releases/tag/3.5.0){: target="_blank"}, released on March 31, 2021.
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+## Product enhancements
+
+-   Now, Codacy allows [configuring who is allowed to ignore files](https://docs.codacy.com/organizations/roles-and-permissions-for-synced-organizations/#change-analysis-configuration) on all supported Git providers. (CY-3883)
+-   Now, Codacy allows [configuring who is allowed to manage patterns](https://docs.codacy.com/organizations/roles-and-permissions-for-synced-organizations/#change-analysis-configuration) on all supported Git providers.  (CY-3822)
```

Comment:
I think these should be together like

```suggestion
-   Now, Codacy allows [configuring who is allowed to manage patterns and ignore files](https://docs.codacy.com/organizations/roles-and-permissions-for-synced-organizations/#change-analysis-configuration) on all supported Git providers. (CY-3883)
```

Reply:
Yeah thought about it as well, I like it!